smt/applications/tests/test_smfck.py

```diff
@@ -80,7 +81,7 @@ def test_smfck(self):
 
             t_error = num / den
 
-            self.assert_error(t_error, 0.0, 2e-1, 2e-1)
+            self.assert_error(t_error, 0.0, 1.0, 1.0)
```

Comment:
You relax the test tolerance to make it pass with scipy cobyla ? Add a comment here to explain that one has to install nlopt to use Cobyla-nlopt to get better performance. Maybe add keep the old line `self.assert_error(t_error, 0.0, 2e-1, 2e-1)` to tell it works with Cobyla from no=lopt.